kinto/plugins/history/__init__.py

```diff
@@ -12,7 +12,14 @@ def includeme(config):
     # Activate end-points.
     config.scan('kinto.plugins.history.views')
 
+    # If StatsD is enabled, monitor execution time of listener.
+    if config.registry.statsd:
+        key = 'plugins.history'
+        listener = config.registry.statsd.timer(key)(on_resource_changed)
+    else:
+        listener = on_resource_changed
```

Comment:
How do you like the "simple assignment before if" instead? Eg

```python
listener = on_resource_changed
if config.registry.statsd:
    key = 'plugins.history'
    listener = config.registry.statsd.timer(key)(on_resource_changed)
```

I know some people prefer this, some people despise it ;)